includes/file-uploader/class-file-uploader.php

```
@@ -157,29 +157,59 @@ public static function mark_tickets_auto_delete_attachments( $type = 'all', $aut
+				$type_param = null;
+			}
+		} else {
+			$type_param = null;
 		}
 
 		$meta_value = $auto_delete ? '1' : '';
 
-
-
-		$select_q = "SELECT pm.post_id, 'auto_delete_attachments' as meta_key, '{$meta_value}' as meta_value from $wpdb->postmeta pm
+		$select_q_base = "SELECT pm.post_id, 'auto_delete_attachments' as meta_key, %s as meta_value from $wpdb->postmeta pm
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$select_q_base'. (undefined)

(UnusedLocalVariable)